contactaccessrights/js/src/access-rights/controllers/access-rights-modal-ctrl.js

```diff
@@ -39,11 +41,14 @@ define([
        * Saves data and closes the modal
        */
       vm.submit = function () {
+        vm.hasSubmit = true;
         $q.all([persistValues('regions'), persistValues('locations')])
```

Comment:
I wonder if you could do the same with simply `.finally()`?
```js
$q.all()
  .then(function () {

  })
  .catch(function () {

  })
  .finally(function () {
    vm.submitting = false;
  });
```
not sure thought, you should test it